tests/e2e/browser/音声詳細.spec.ts

```diff
@@ -74,3 +74,52 @@ test("詳細調整欄のコンテキストメニュー", async ({ page }) => {
   await expect(page.getByText("ニヒャク")).toBeVisible();
   await expect(page.getByText("ヨン")).toBeVisible();
 });
+
+test("アクセント区間全体の値変更", async ({ page }) => {
+  const otherSliderValue = process.env.CI ? "0.106" : "0.064";
+
+  await navigateToMain(page);
+  await page.waitForTimeout(100);
+
+  const textField = page.getByRole("textbox", { name: "1行目" });
+  await textField.click();
+  await textField.fill("1234");
+  await textField.press("Enter");
+
+  await page.getByText("長さ").click();
+  await page.waitForTimeout(1000);
+
+  const moraTable = page.locator(".mora-table").last();
+  await moraTable.hover({
+    // Alt 押下中はスライダー以外にも当たり判定があることを確認するため
+    position: { x: 20, y: 20 },
+  });
+  await page.keyboard.down("Alt");
+
+  // Alt 押下中は hover 中のアクセント区間の他モーラの値ラベルも表示される
+  await expect(page.getByText(otherSliderValue)).toBeVisible();
```

Comment:
この区間のテストが`Alt 押下中は hover 中のアクセント区間の他モーラの値ラベルも表示される`かどうかをチェックするための準備と確認だと思うので、これらを１つにしてコメントを書くとわかりやすいのかなと思いました！
こんな感じで･･･！
```suggestion
  // Alt 押下中は hover 中のアクセント区間の他モーラの値ラベルも表示される
  await moraTable.hover({
    // Alt 押下中はスライダー以外にも当たり判定があることを確認するため
    position: { x: 20, y: 20 },
  });
  await page.keyboard.down("Alt");
  await expect(page.getByText(otherSliderValue)).toBeVisible();
```